l10n_it_fatturapa_out_wt/wizard/wizard_export_fatturapa.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+from odoo.tools.translate import _
+from odoo.exceptions import Warning as UserError
+from odoo.addons.l10n_it_fatturapa.bindings.fatturapa_v_1_2 import (
+    DatiRitenutaType,
+    AltriDatiGestionaliType,
+)
+
+
+class WizardExportFatturapa(models.TransientModel):
+    _inherit = "wizard.export.fatturapa"
+
+    def setDatiGeneraliDocumento(self, invoice, body):
+        res = super(WizardExportFatturapa, self).setDatiGeneraliDocumento(
+            invoice, body)
+        ritenuta_lines = invoice.withholding_tax_line_ids.filtered(
```

Comment:
@sergiocorato  grazie mille per questa PR. testato e pare funzionare. come suggerimento, metterei un warning se la ritenuta non è configurata. con questo "invoice.withholding_tax_line_ids.filtered(..." c'è il rischio che se la ritenuta NON è configurata non viene esportata, cosa che non credo debba succedere.

Cmq ho testato il flusso e ho testato il file xml nel sito dell'agenzia e il test da buon esito. 

Gazie, giacomo

Reply:
ciao @sergiocorato, da runbot ho testato il flusso e verificato il fil xml con lo strumento messo a disposizione della ade. Sembra tutto apposto. Grazie!

Reply:
@marcelofrare se il test funzionale è andato a buon fine puoi approvare la PR indicando la cosa per favore?

Basta andare in alto a destra, fai clic su "Files changed"

![immagine](https://user-images.githubusercontent.com/227940/52789721-f19d6e00-3064-11e9-92c7-ec09ae7d7e26.png)

quindi fai clic sul pulsante verde «Review changes» sempre in altro a destra, inserisci il commento, seleziona "Approve" e infine fai clic su «Submit review».

![immagine](https://user-images.githubusercontent.com/227940/52789769-1b569500-3065-11e9-9b67-5a6c7d676e0f.png)